host/host_aix.go

```diff
@@ -105,12 +107,12 @@ func parseUptime(uptime string) uint64 {
 				return 0
 			}
 		}
-	case "hr,", "hrs,":
+	case "hr,", "hrs,", "hr", "hrs":
```

Comment:
This case also needs a subcase for mins to be in ut[5], with appropriate protections.